src/app/containers/App/App.test.jsx

```diff
@@ -58,6 +58,7 @@ describe('App', () => {
     expect.assertions(4);
     expect(route.getInitialData).not.toHaveBeenCalled();
     expect(reactRouterConfig.renderRoutes).toHaveBeenCalledTimes(1);
+    // commit note
```

Comment:
There are a few fails in this file, which all revolve around changes made to the App.jsx state object. 

It [previously explicitly declared each expected value](https://github.com/bbc/simorgh/pull/7825/files#diff-075d3e454c8d5028eecfaf8c11b80de1L78-L92)

Now it is [spreading the contents of those objects](https://github.com/bbc/simorgh/pull/7825/files#diff-075d3e454c8d5028eecfaf8c11b80de1R42-R47)

So an assertion like the one below on `ln62` could previously constructively be made, but now the state object in `renderRoutes` is far more complex. Taking in the whole `getInitialData` object (for example) [rather than deconstructing it](https://github.com/bbc/simorgh/pull/7825/files#diff-075d3e454c8d5028eecfaf8c11b80de1L20-L27) as previously. 

Two options here: make the tests assert against the more complex objects it's being called with (img below), or revert the changes in App.jsx to deconstruct, rather than spread, the objects.

![Screenshot 2020-09-23 at 16 36 28](https://user-images.githubusercontent.com/9042672/94035630-3b127b00-fdbb-11ea-9027-b40147261643.png)